build/build.py

```diff
@@ -11,6 +11,19 @@
 import tarfile
 import zipfile
 
+__version = "0.56.0.0"
+
+def __globalVariables( buildDir ) :
+
+	return {
+		"buildDir" : buildDir,
+		"jobs" : multiprocessing.cpu_count(),
+		"path" : os.environ["PATH"],
+		"version" : __version,
+		"platform" : "osx" if sys.platform == "darwin" else "linux",
```

Comment:
We need to remember to move to `macos` at some point when we're ready to update `gaffer` too.

Reply:
What does the update in gaffer entail? Seems like maybe I should change it now?

Reply:
I think there is:
https://github.com/GafferHQ/gaffer/blob/3d42aa2051f2a07932b02724ccc7b138921b5d45/config/installDependencies.sh#L50
and:
https://github.com/GafferHQ/build/blob/cb9e4594879a1f49f551bb1ef1061d932882638a/build.py#L159
May be others

Reply:
Well, this is going to be gafferDependencies-1, and we'll need to change Gaffer to pick it up anyway, so how about I just do it now?

Reply:
Sounds good to me!